parquet/src/test/java/org/apache/iceberg/parquet/TestVariantReaders.java

```diff
@@ -329,7 +331,7 @@ public void testFixedLengthByteArray() {
   public void testShreddedObject() throws IOException {
     GroupType fieldA = field("a", shreddedPrimitive(PrimitiveTypeName.INT32));
     GroupType fieldB = field("b", shreddedPrimitive(PrimitiveTypeName.BINARY, STRING));
-    GroupType objectFields = objectFields(fieldA, fieldB);
+    GroupType objectFields = objectFields(List.of(fieldA, fieldB));
```

Comment:
Please revert this change, I don't think it is necessary.

Reply:
Reverted